joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

pitch_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: pitch_joint
    pid: {p: 0.5, i: 0.0, d: 0.05}

yawl_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: yawl_joint
    pid: {p: 0.5, i: 0.0, d: 0.05}